pegasus/routes/hoc_routes.rb

```diff
@@ -49,7 +49,7 @@
 get '/api/hour/begin_learn/:code' do |code|
   only_for ['code.org', 'csedweek.org', partner_sites].flatten
   pass unless tutorial = DB[:tutorials].where(code: code).first
-  launch_tutorial(tutorial, track_learn: true)
+  redirect tutorial[:url], 302
```

Comment:
@Hamms I know this function is no longer called, but since this didn't work the way we wanted, should we revert this change on staging?